WooCommerce/Classes/ViewRelated/ReusableViews/TextFieldContentAlignment.swift

```diff
@@ -0,0 +1,22 @@
+import UIKit
+
+/// Horizontal alignment for content (text or placeholder) in a text field.
+enum TextFieldContentAlignment {
```

Comment:
⚠️  As you implied, the name is a bit confusing. 😬 But I think what would be more valuable is to explain the use cases of this `enum`. Why and when would we want to use this instead of just `NSTextAlignment`? What do you think about adding a comment about that?

Reply:
I renamed this to `TextFieldTextAlignment` and updated the comment in https://github.com/woocommerce/woocommerce-ios/pull/2442/commits/1e378d324ae0088c28309c6471d46615905e6988 - lemme know if any part can still be improved!

Reply:
Awesome! The description and the unit tests really help!